### PR TITLE
Modified README.md

### PR DESCRIPTION
**Pull request by kaushik-p9**
There were a bunch of grammatical errors in the readme.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Python bindings.
 
 ###  1. Introduction
 
-The mlpack website can be found at http://www.mlpack.org and contains numerous
+The mlpack website can be found at http://www.mlpack.org and it contains numerous
```
Comment (ShikharJ) [inline README.md]: I wouldn't say this is a grammatical error.

Comment (kaushik-p9) [reply README.md]: Lol, you are right @ShikharJ. Do you mind telling me which projects need help in mlpack?

Comment (rcurtin) [reply README.md]: I agree, technically it's ok as-is, but I'm also happy to merge both fixes.  The second one is definitely an issue. :)

Comment (kaushik-p9) [reply README.md]: @rcurtin could you please tell me some projects that I can work on, other than the bugs? :)

**Comment by Wenhao-H** [comment]
@kaushik-p9 I just posted an issue at #1542. You can try working on that if you are interested.

**Comment by rcurtin** [comment]
This has two approvals, so I'll go ahead and merge it now.  Thanks again. :+1:

**Pull request merged**